src/pubsub/peer-streams.js

```diff
@@ -123,11 +133,13 @@ class PeerStreams extends EventEmitter {
     // - transformed with length-prefix transform
     this._inboundAbortController = new AbortController()
     this._rawInboundStream = stream
+    // @ts-ignore - abortable returns AsyncIterable and not a DuplexIterableStream
```

Comment:
i think the type `DuplexIterableStream` should be improved because its just an object and not a iterable or a stream :)

Reply:
It was changed to a MuxedStream as it is what it should be